apis/sgv2-restapi/CONFIGURATION.md

```diff
@@ -0,0 +1,22 @@
+# Configuration Guide
+
+> **IMPORTANT:** Please check the [Stargate Common Configuration](../sgv2-quarkus-common/CONFIGURATION.md) for properties shared between all Stargate V2 APIs.
+
+## Stargate REST API Configuration
```

Comment:
is there no other restapi prop in the whole project?

Reply:
Let me go check, good point.

Reply:
I don't see any, looking at `application.yaml`s under `sgv2-restapi`. There is at least one for SGv1 REST API (in-coordinator one) for inclusion of RESTv1 but nothing for SGv2 REST portion that I remember.

I'll keep this in mind if I mind something and make sure to update this page.

Reply:
Make sure to also update the README, which currently says:

```
The REST API currently does not define any additional properties beyond these common properties.
```

Reply:
> Make sure to also update the README, which currently says:
> 
> ```
> The REST API currently does not define any additional properties beyond these common properties.
> ```

Changed. Thanks!